app/styles/_core.less

```diff
@@ -532,6 +539,38 @@ label.checkbox {
   }
 }
 
+.osc-secrets-form {
+  .osc-secret-actions {
+    font-size: 11px;
```

Comment:
wanted to match the font size we are using for `Add GitHub webhook`. Shall I set that to the default one as well ?